Wbooks/Main/MainViewController.swift

```diff
@@ -28,7 +29,8 @@ final class MainViewController: UIViewController {
     @objc private func loginButtonTapped() {
         let controller = LibraryController()
         controller.modalPresentationStyle = .fullScreen
-        present(controller, animated: true)
+        navigationController?.setNavigationBarHidden(false, animated: true)
+        navigationController?.pushViewController(controller, animated: true)
```

Comment:
Instead of wrapping the MainViewController in a NavigationController, let's do that with the LibraryViewController:
```
let controller = UINavigationController(rootViewController: LibraryController())
controller.modalPresentationStyle = .fullScreen
present(controller, animated: true)
```